README.md

```diff
@@ -12,6 +12,8 @@
 
 <h4 id="top" align="center">Formerly known as "Pulse"</h1>
 
+[SPV Wallet GitBook](https://bsvblockchain.gitbook.io/docs)
```

Comment:
IMHO this link should be introduced with some text, e.g. what it is and what an user can find there.